renv.lock

```diff
@@ -1,6 +1,6 @@
 {
   "R": {
-    "Version": "4.1.2",
+    "Version": "4.1.0",
```

Comment:
I think it would be better to keep using the latest R version. You probably need to update locally and then run `renv::snapshot()` again.

Reply:
ok